Dockerfile

```diff
@@ -1,10 +1,20 @@
+# Configurable arguments
 ARG BUILD_ENV=default
-FROM golang:1.16-buster AS base
+ARG GOLANG_VERSION=1.16-buster
+
+# Install build time dependencies
+FROM golang:${GOLANG_VERSION} AS base
+
+ARG APT_INSTALL_EXTRA_DEPS
+ENV APT_INSTALL_EXTRA_DEPS=$APT_INSTALL_EXTRA_DEPS
```

Comment:
Do we need this ENV or can we use the ARG in the install command?

Reply:
nope